### PR TITLE
allow back button to close new tabs

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/715106103902962/1165288656835289
Tech Design URL:
CC:

**Description**:

Allows back button to open new tabs opened as follows:

* Open in new tab context menu option (on a link on the page)
* Opened by the page

**Steps to test this PR**:
1. Go to duckduckgo.com and change the settings to open links in a new window
1. Perform a search
1. Click on a link - should open a new tab
1. Click back - new tab should close
--
1. Click on another link
1. Use the tab switcher to close the opening tab
1. Go back to opened page, but button should not close page
--
1. Go to https://news.ycombinator.com
1. Open a link using the open in new tab menu option
1. Perform above tests
--
1. Open a new link using open in background menu option
1. Use tab switcher to go to new tab
1. Should not be able to close tab using back button

**Device Testing**:

* [x] iPhone X
* [x] iPad

**OS Testing**:

* [x] iOS 10
* [x] iOS 11
* [x] iOS 12
* [x] iOS 13

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Review by subsymbolic — APPROVED** [review]
LGTM!

**Comment by subsymbolic** [comment]
Do we want the "Open in background tab" option to have this behavior too?

**Comment by brindy** [comment]
Thanks for the review - waiting for the product review to approve before merging.

re: Open in background ... no because it's kind of disconnected experience.  If you've gone to the tab switcher to select the tab then it becomes ambiguous to what "back" means then.

**Pull request merged**
